tests/view/test_reservations.py

```diff
@@ -16,6 +18,27 @@
 
 """
 
+def test_reservation_list(client, db_session):
+    employee = factories.EmployeeFactory(
+        company=factories.CompanyFactory()
+    )
+
+    with client.session_transaction() as session:
+        session["user_id"] = employee.id
+
+    #Cheated reservations
+    factories.ReservationFactory()
```

Comment:
@ebercamargo `factories.ReservationFactory.create_batch(size=4)` should work

Reply:
I put it that way, despite being equal in the tests. I noticed that other methods affect the test of this, so I had to start counter ID in 3.